src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,16 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Why set in normalize.scss f-f: Roboto and f-s: 12 if you set in calendar__day f-f Arial and f-s: 30px?
The f-f: Roboto and f-s: 12 is not used anywhere else.